README.md

```diff
@@ -6,6 +6,18 @@ instances, attribute hashes, and stubbed objects), and support for multiple
 factories for the same class (`user`, `admin_user`, and so on), including factory
 inheritance.
 
+### Transitioning from factory\_girl\_rails?
+
+Transitioning is a two step process:
+
+1) Replace `factory_girl_rails` with `factory_bot_rails` in your Gemfile
+    and `bundle install`
+
+2) Rename all `FactoryGirl` to `FactoryBot` and `factory_girl` to `factory_bot`
+    in your codebase. This command may be helpful:
+
+    `ag -0l FactoryGirl spec | xargs -0 sed -i 's/FactoryGirl/FactoryBot/g'`
```

Comment:
I wonder if we should specify that this only happen in `spec` - specifically, I'm thinking about situations like `rake dev:prime` where FG is used that would be missed.

Reply:
Thoughts on backing this out to the top-level indentation and using triple-backticks?

```
```sh
```